packages/bot-web-ui/src/pages/server-side-bot/add-bot/descriptions/strategy-description.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Text } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { TStrategyDescription } from '../types';
+
+const StrategyDescription = observer(({ item, font_size }: TStrategyDescription) => {
+    const { ui } = useStore();
+    const { is_dark_mode_on } = ui;
+    const class_name = item?.className ?? '';
+    switch (item.type) {
+        case 'text': {
+            const class_names = classNames(`ssb-add__description__content ${class_name}`);
+            return (
+                <>
+                    {item?.content?.map((text: string) => (
+                        <div className={class_names} key={text}>
+                            <Text size={font_size} dangerouslySetInnerHTML={{ __html: text }} />
+                        </div>
+                    ))}
+                </>
+            );
+        }
+        case 'text_italic': {
+            const class_names = classNames(`ssb-add__description__content italic ${class_name}`);
+            return (
+                <>
+                    {item?.content?.map((text: string) => (
+                        <div className={class_names} key={text}>
+                            <Text size={font_size} dangerouslySetInnerHTML={{ __html: text }} />
```

Comment:
```suggestion
                            <Text size={font_size} dangerouslySetInnerHTML={{ __html: text }} />
```
wow man the name of the prop itself is `dangerouslySetInnerHTML` 😆  is there no better way to do this

Reply:
Let us see if we can get rid of it @yashim-deriv 🙇

Reply:
This has been resoved 👍